fixtures/t0109-dnslink.yml

```diff
@@ -0,0 +1,6 @@
+dnslinks:
```

Comment:
Should we add a JSON schema for the config files?

```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "type": "object",
  "properties": {
    "dnslinks": {
      "type": "object",
      "additionalProperties": {
        "type": "object",
        "properties": {
          "subdomain": {
            "type": "string"
          },
          "path": {
            "type": "string"
          }
        },
        "required": ["subdomain", "path"],
        "additionalProperties": false
      }
    }
  },
  "additionalProperties": false
}
```

Then, there are editor plugins that could verify the config files for us if we provide the path to the schema like this:

```suggestion
# yaml-language-server: $schema=PATH_TO_SCHEMA_DOT_JSON
dnslinks:
```

Reply:
I love it, added